Modules/System/SharePoint/src/model/file/SharePointFile.Table.al

```diff
@@ -0,0 +1,88 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about file entity.
+/// </summary>
+table 9100 "SharePoint File"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint File';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; "Unique Id"; Guid)
```

Comment:
Same, call this Id

Reply:
Hi, as agreed we will keep the SharePoint naming.